.github/workflows/run-tests.yml

```diff
@@ -18,10 +18,18 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      
+      - uses: docker/setup-buildx-action@v1
+        id: buildx
+        with:
+          install: true
+
+      - name: Expose GitHub Runtime
+        uses: crazy-max/ghaction-github-runtime@v1
```

Comment:
What is this action used for?

Reply:
Friends gave me the hint how to better utilize buildkit in github actions. You can use buildx in combination with github actions caching (`--cache-from=type=gha` and `--cache-to=type=gha`) 
Therefore you must enable buildkit and expose the runtime so `docker buildx build` can access the variables inside the run step.